### PR TITLE
Implement distutils.strtobool in class fields.StringBoolean

### PR DESCRIPTION
**Pull request by DemiBSel**
To remove dependency on distutils as suggested in #203.

### CODE REVIEW

**Comment by betodealmeida** [comment]
Thanks for working on this, @DemiBSel! I like the approach of adding it as a static method in `StringBoolean`.

Do you want to add yourself to `AUTHOR.rst`, as a contributor?

**Comment by DemiBSel** [comment]
Sure thank you ! Kind of messed up the commit, guess I shouldn't have tried to use commit --amend for this.

**Comment by DemiBSel** [comment]
Thanks for your review ! Don't merge yet, I will check the coverage soon.

File: tests/test_fields.py [hunk 1]
```diff
@@ -263,6 +265,9 @@ def test_string_boolean() -> None:
     """
     assert StringBoolean().parse("TRUE") is True
     assert StringBoolean().parse("FALSE") is False
+    with pytest.raises(ValueError) as expected_err:
+        StringBoolean().parse("B")
+    assert str(expected_err.value) == "invalid truth value b"
```
Comment (betodealmeida) [inline tests/test_fields.py:270]: Thank you!

**Review by betodealmeida — APPROVED** [review]

**Pull request merged**